src/react/hooks/juicebox/useProjectBalance.ts

```diff
@@ -0,0 +1,18 @@
+import { useJBContractContext } from "src/react/contexts";
+import { useJbSingleTokenPaymentTerminalStoreBalanceOf } from "../../generated/hooks";
+
+/**
+ * Returns the balance of a single token payment terminal store for a given project.
+ *
+ * @returns The balance of the payment terminal store as a `bigint`.
+ */
+export function useProjectBalance(): bigint {
+  const { contracts } = useJBContractContext();
+  const { primaryTerminalEthStore } = contracts;
+
+  const balance = useJbSingleTokenPaymentTerminalStoreBalanceOf({
+    address: primaryTerminalEthStore.data,
+  });
+
+  return balance.data ?? 0n;
```

Comment:
suggestion: just return `balance`, so the loading and error states are returned

Reply:
Done